js/apps/admin-ui/src/components/users/UserDataTable.tsx

```diff
@@ -391,6 +424,11 @@ export function UserDataTable() {
             displayKey: "firstName",
             cellFormatters: [emptyFormatter()],
           },
+          {
+            name: "Enable",
+            displayKey: "Enable",
```

Comment:
This needs to refer to a label's key, not a verbatim of what the displayed value should be. Make sure to add a label as well.